plugins/Live/templates/_actionsList.twig

```diff
@@ -1,39 +1,39 @@
 {% set previousAction = false %}
 {% for actionGroup in actionGroups %}
     {% if actionGroup.pageviewAction is not empty %}
-    {{ postEvent('Live.renderAction', actionGroup.pageviewAction, previousAction, visitInfo) }}
```

Comment:
ℹ️ This file contains only whitespace (indentation / extra lines) refactoring because I couldn’t follow.